Store/COSQLiteStore.m

```diff
@@ -124,14 +124,6 @@ - (void)dealloc
         [db_ close];
         db_ = nil;
     });
-
-#if !(TARGET_OS_IPHONE)
```

Comment:
Should change to `#if defined(GNUSTEP)`
and the start of the comment should be changed to `We are using deployment target 10.8, so ARC will manage libdispatch objects automatically. For GNUstep....`